lldb/source/Plugins/Process/gdb-remote/GDBRemoteCommunicationServerPlatform.cpp

```diff
@@ -159,6 +159,40 @@ GDBRemoteCommunicationServerPlatform::GDBRemoteCommunicationServerPlatform(
 GDBRemoteCommunicationServerPlatform::~GDBRemoteCommunicationServerPlatform() =
     default;
 
+void GDBRemoteCommunicationServerPlatform::Proc(
+    const lldb_private::Args &args) {
+  if (!IsConnected())
+    return;
+
+  if (args.GetArgumentCount() > 0) {
+    lldb::pid_t pid = LLDB_INVALID_PROCESS_ID;
+    std::optional<uint16_t> port;
+    std::string socket_name;
+    Status error = LaunchGDBServer(args,
+                                   "", // hostname
+                                   pid, port, socket_name);
+    if (error.Success())
+      SetPendingGdbServer(pid, *port, socket_name);
```

Comment:
What happens with the implicit `else` part of this, does `GetPacketAndSendResponse` somehow fail immediately?

Reply:
This code is just moved from lldb-platform.cpp 
GetPacketAndSendResponse() ignores `error` value. GetPacketAndSendResponse() must set the flag `quit` or `interrupt` and update `error`.